packages/lib/constants.ts

```diff
@@ -151,7 +151,6 @@ export const BOOKER_NUMBER_OF_DAYS_TO_LOAD = parseInt(
 export const CLOUDFLARE_SITE_ID = process.env.NEXT_PUBLIC_CLOUDFLARE_SITEKEY;
 export const MINIMUM_NUMBER_OF_ORG_SEATS = 30;
 export const ORG_SELF_SERVE_ENABLED = process.env.NEXT_PUBLIC_ORG_SELF_SERVE_ENABLED === "1";
-export const ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE = 2;
```

Comment:
Instead of Removing the code, it would make sense to set it to 0 and get it working.

it would be useful if we want to tweak the limit later again

Reply:
I have set `ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE` to 0. Tested it. Works fine.